source/Junkership.js

```diff
@@ -82,49 +83,49 @@ export default class Junkership extends Pixi.Sprite {
             }
         }
 
-        if(Keyb.isJustDown("1")) {
-            this.changePowerUp(new PeaShoota)
-        }
-
-        if(Keyb.isJustDown("2")) {
-            this.changePowerUp(new TriShoota)
-        }
-
-        if(Keyb.isJustDown("3")) {
-            this.changePowerUp(new FiveShoota)
-        }
-
-        if(Keyb.isJustDown("4")) {
-            this.changePowerUp(new RapidFire)
-        }
-
-        if(Keyb.isJustDown("5")) {
-            this.changePowerUp(new SprayShot)
-        }
-
-        if(Keyb.isJustDown("6")) {
-            this.changePowerUp(new SuperSprayShot)
-        }
-
-        if(Keyb.isJustDown("7")) {
-            this.changePowerUp(new CrazySprayShot)
-        }
-
-        if(Keyb.isJustDown("8")) {
-            this.changePowerUp(new VertSprayShot)
-        }
-
-        if(Keyb.isJustDown("9")) {
-            this.changePowerUp(new VertShoota)
-        }
-
-        if(Keyb.isJustDown("0")) {
-            this.changePowerUp(new RapidSprayShot)
-        }
-
-        if(Keyb.isJustDown("-")) {
-            this.changePowerUp(new BFG)
-        }
+//        if(Keyb.isJustDown("1")) {
+//            this.changePowerUp(new PeaShoota)
+//        }
+//
+//        if(Keyb.isJustDown("2")) {
+//            this.changePowerUp(new TriShoota)
+//        }
+//
+//        if(Keyb.isJustDown("3")) {
+//            this.changePowerUp(new FiveShoota)
+//        }
+//
+//        if(Keyb.isJustDown("4")) {
+//            this.changePowerUp(new RapidFire)
+//        }
+//
+//        if(Keyb.isJustDown("5")) {
+//            this.changePowerUp(new SprayShot)
+//        }
+//
+//        if(Keyb.isJustDown("6")) {
+//            this.changePowerUp(new SuperSprayShot)
+//        }
+//
+//        if(Keyb.isJustDown("7")) {
+//            this.changePowerUp(new CrazySprayShot)
+//        }
+//
+//        if(Keyb.isJustDown("8")) {
+//            this.changePowerUp(new VertSprayShot)
+//        }
+//
+//        if(Keyb.isJustDown("9")) {
+//            this.changePowerUp(new VertShoota)
+//        }
+//
+//        if(Keyb.isJustDown("0")) {
+//            this.changePowerUp(new RapidSprayShot)
+//        }
+//
+//        if(Keyb.isJustDown("-")) {
+//            this.changePowerUp(new BFG)
+//        }
```

Comment:
Boo! Commented out code! Let's instead either:

a) Delete it; we can always revert to this functionality through Git.

b) Wrap it in a URL Query, so we enable these debug keys when the URL has `?debug=1` at the end of it.
https://github.com/mocsarcade/starjunk/wiki/Code:Cookbook#how-to-get-a-query-from-the-url